README.md

```diff
@@ -68,6 +69,21 @@ $ sudo find / -name qmake
 ## Run:
 ``` ./start.sh [options] [script file name] ```
 
+## Documentation generation
+### Download and install [ldoc](stevedonovan.github.io/ldoc/manual/doc.md.html)
+```
+sudo apt install luarocks
+sudo luarocks install penlight
```

Comment:
@VVeremjova, please take a look at the [pxory setup](https://github.com/luarocks/luarocks/wiki/LuaRocks-through-a-proxy) and [luarocks cache remove](https://github.com/torch/torch7/issues/303)
I very hope it helps!